democracy_club/templates/privacy.html

```diff
@@ -7,38 +7,78 @@ <h1>Privacy &amp; Terms</h1>
 
-* We will not sell or distribute your email address or other personal
-information.
-* We will gladly show you the personal data we store about you in
-order to run the website.
 
-We may use your email address to contact you about your use of the site (for
-example, to ask for feedback) or with occasional announcements about this
-project or similar Democracy Club projects.
+## Privacy, in detail
 
-### Some more details
+###Why do we collect data?
 
-As a visitor to the site, we store your IP address. This information is
-accessible to administrators of the site.
+The law says we must have a ‘lawful purpose’ to collect personal data. We collect and process personal information either because we have your explicit consent to do so or because we have a legitimate interest in improving our services. 
 
-If you login to the site, then we store your login details (obviously). We
-also (obviously) store the details of any data you contribute to the site.
-This information is accessible to administrators of the site.
-
-We also use Google Analytics to aggregate and track data about usage of the
-site; you can use a browser plugin to opt out of having data collected by
-Google Analytics.
+Democracy Club CIC runs several digital services whose purpose is to increase democratic engagement, as well as secondary services that indirectly serve those aims, such as a crowdsourcing database, an email list and an online chat space. These services or tools are made more useful or can only exist as a result of personal data being shared with us.
 
 If you have any questions about this privacy policy, please do [email
 us](mailto:hello@democracyclub.org.uk).
 
+###What data do we collect?
+
+When you use any of our services, we may store your IP address ([a unique identifier for your device or internet connection](https://simple.m.wikipedia.org/wiki/IP_address)) in order to understand how our websites are used and to help with some parts of our server administration. personalise the service to you. 
+
+If you sign up to any of our services, such as a mailing list or election reminders, we collect the data you provide so that we can keep in touch and provide you with relevant information.
+
+If you sign up to help crowdsource data, we will store your username and email address. The edits you make will be publicly recorded alongside your username, which, depending on the username you choose, may allow you to be identified personally. If you have chosen an anonymous username, Democracy Club administrators will be able to see the email address linked to it.
+
+Several of our services provide information to you based on your postcode. We store your postcode so we can total up user numbers in an area to help us focus our efforts. For example, we made [this table](https://democracyclub.github.io/wheredoivote-usage-may-2018/data/wheredoivote-usage-may-2018) of users in a council area.
```

Comment:
Can we tweak this line to be explicit that the data we produce is anonymised. e.g:

> We store your postcode so we can produce anonymised statistics. For example..

Will leave wording with you